plugins/googleanalytics/client/Settings.tsx

```diff
@@ -87,15 +90,40 @@ function GoogleAnalytics() {
           )}
           border={false}
         >
-          <Input placeholder="G-XXXXXXXXX1" {...register("measurementId")} />
+          <Input
+            placeholder="G-XXXXXXXXX1"
+            {...register("measurementId", { required: true })}
+          />
         </SettingRow>
 
-        <Button type="submit" disabled={formState.isSubmitting}>
-          {formState.isSubmitting ? `${t("Saving")}…` : t("Save")}
-        </Button>
+        <Flex gap={8}>
```

Comment:
Can we just switch the order of these and align to the right, so that it's 

[Disconnect] [Save]

with save on the far right, remaining primary. I think this will be more natural